src/che-theia-ssh-plugin-backend-plugin.ts

```diff
@@ -0,0 +1,152 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as theia from '@theia/plugin';
+import { CheService, SshKeyPair } from "./common/ssh-protocol";
```

Comment:
minor issue but in the code sometimes `"` is used, sometimes `'`

Reply:
fixed